supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -90,6 +90,17 @@
 
 *See also*:
 
+[discrete]
+[[scriptlet-directive]]
+==== image:images/yes.png[yes] scriptlet directive (noun)
+*Description*: A _scriptlet_ is a fragment of code in a SPEC file, such as a shell script, that runs during build time ("build scriptlet") or transaction time ("transaction scriptlet") to change the user’s system. Build scriptlets are more limited than transaction scriptlets because build scriptlets do not receive arguments and can only be shell scripts.
```

Comment:
ok that there's no mention of a "directive" in desc?

Reply:
+1

Reply:
Thank you for pointing that out, checked with SMEs -> changing it to lowercase in all occurrences